content/post/22-05-19-progress-report-34.md

```diff
@@ -0,0 +1,470 @@
+
+Currently only the PietyVend takes advantage of this; it offers occult items to any Chaplains that have... fallen from grace.
+
+{{< newsection >}}
+### Station Map Posters
+*Contributed by Emisse*
+
+Most existing stations now have map posters, which give a tiny overview of the station's general layout. They can be found in various places on supported stations. The colors represent departments, such as red for security and blue for medical.
+
+{{< imgw "images/post/pr_34/posters" >}}
+
+{{< newsection >}}
+### Atmospheric Technician
+*Contributed by Emisse*
+
+Space Station 14 has a new job from SS13 this month in the form of the Atmospheric Technician, an engineer dedicated to working with the maze of pipes and gas pumps that make up the station's atmospheric system. 
+
+TODO Pic?
+
+{{< newsection >}}
+### Merge ID Accesses
+*Contributed by 20kdc*
+
+In a recent survey, frustration among Heads of Personnel has been down across the board after a change to the way access works. Doors and windoors now check all of the spots an ID could be in before denying you access - meaning that the Head of Personnel can now open the windoor to their desk and hand an ID card across without having to drop it on the floor first.
+
+{{< newsection >}}
+### Vehicles
+*Contributed by Rane, & Veritius*
+
+A shipment of vehicles have arrived at the station, and security has already received several reports of traffic violations. Stop running over the Clown before we have a lawsuit on our hands.
+
+Here's a list of every vehicle you can find or order so far:
+
+- ATVs
+- Janicart
+- Secways
+- Syndicate Segways
+
+{{< video-audio "/video/pr_34/secway.mp4" >}}
+
+{{< newsection >}}
+### Mime Powers
+*Contributed by Rane*
+
+The Mime now sports the uncanny and somewhat disturbing ability to create a temporary invisible wall, blocking the passage of objects and station personnel as well as a regular wall.
+
+However, like all superheroes, the Mime must have a weakness. The Mime now starts with a vow of silence and loses their wall ability if the vow is broken. 
+
+TODO Vid?
+
+{{< newsection >}}
+### Beds, Stasis Beds, and Beyond
+*Contributed by Rane*
+
+A new study shows that a bit of bedrest can do wonders for a crewmember's health, and medical beds are even better. Additionally, science can research new stasis beds that will greatly slow down the metabolism of any occupants. Now you can make your bad trip last even longer!
+
+{{< newsection >}}
+### Zombies
+*Contributed by EmoGarbage404*
+
+BRAAAAAAAAAAAAAAINS...
+
+TODO: Pic of zombocalypse
+
+{{< newsection >}}
+### Admin Notes
+*Contributed by DrSmugleaf*
+
+Admin notes are the latest tool from SS13 in our anti-greytide arsenal, allowing admins to record players' heinous deeds on their permanent record. Be careful,  just because you don't get banned doesn't mean your actions aren't being noted!
```

Comment:
```suggestion
Admin notes are the latest tool from SS13 in our anti-greytide arsenal, allowing admins to record players' heinous deeds on their permanent record. Be careful, just because you don't get banned doesn't mean your actions aren't being noted!
```